doc/api/stream.md

```diff
@@ -2453,9 +2453,14 @@ await Readable.from([1, 2, 3, 4]).take(2).toArray(); // [1, 2]
 added:
   - v17.5.0
   - v16.15.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/48102
+    description: the `asIndexedPairs` property has been deprecated and will be
```

Comment:
```suggestion
    description: Using the `asIndexedPairs` property emits a runtime warning that it will be
```
```suggestion
    description: the `asIndexedPairs` property has been deprecated and will be
```